packages/ndla-ui/src/Resource/BlockResource.stories.mdx

```diff
@@ -47,12 +52,76 @@ export const Template = (args) => <BlockResource {...args} />;
         alt: '',
       },
       resourceTypes: [{ id: 'urn:resourcetype:learningPath', name: 'Læringssti' }],
-      tags: ['tag', 'tag', 'tag', 'tag'],
+      tags: tags,
     }}>
     {Template.bind({})}
   </Story>
 </Canvas>
 
-### Props
+## Props
 
 <ArgsTable story="BlockResource" />
+
+## Varianter
+
+### Blokkressurs med bilde
+
+<Canvas>
+  <BrowserRouter>
+    <BlockResource
+      id="2ae58e7d-ea25-49f0-b2a5-1f11830f8c04"
+      key={'blockResource'}
+      title="Helt Vanlig Tittel"
+      resourceTypes={[{ id: 'urn:resourcetype:subjectMaterial', name: 'Fagstoff' }]}
+      tags={tags}
+      description={'Dette er for eksempel en fagbeskrivelse! Dersom den er for lang vil den bli forkortet'}
+      resourceImage={resourceImage}
+      link={''}
+      menuItems={menuItems}
+    />
+  </BrowserRouter>
+</Canvas>
+
+### Blokkressurs uten bilde
+
+<Canvas>
+  <BrowserRouter>
+    <BlockResource
+      id="2ae58e7d-ea25-49f0-b2a5-1f11830f8c04"
+      key={'blockResource'}
+      title="Helt Vanlig Tittel"
+      resourceTypes={[{ id: 'urn:resourcetype:subjectMaterial', name: 'Fagstoff' }]}
+      tags={tags}
+      description={'Dette er for eksempel en fagbeskrivelse! Dersom den er for lang vil den bli forkortet'}
+      resourceImage={{
+        src: '',
+        alt: '',
+      }}
+      link={''}
+      menuItems={menuItems}
+    />
+  </BrowserRouter>
+</Canvas>
+
+#### Ulike ressurstyper:
+
+<ul>
+  <li>
+    <strong> Læringssti:</strong> urn:resourcetype:learningPath
+  </li>
+  <li>
```

Comment:
Syntax for identisk liste i markdown er `- **Bold tekst:** vanlig tekst`. Tips til neste gang man skal lage lister 😄 

Alle ressurstypene finnes også eksportert fra /ndla-ui/src/model/ContentType.ts